frontend/src/shared/modules/filters/components/Filter.vue

```diff
@@ -10,20 +10,27 @@
     <div class="flex items-center flex-wrap">
       <template v-for="(filter, fi) of filterList" :key="filter">
         <!-- Operator -->
-        <div
-          v-if="fi > 0"
-          class="border text-xs border-gray-100 rounded-md shadow uppercase
-          h-8 flex font-medium items-center py-1 px-2 bg-white cursor-pointer hover:bg-gray-100 transition mr-4 mb-2"
-          @click="switchOperator"
+        <el-tooltip
+          effect="dark"
+          :content="`${filters.relation} → ${filters.relation === 'and' ? 'or' : 'and'}`"
+          placement="top"
         >
-          {{ filters.relation }}
-        </div>
+          <div
+            v-if="fi > 0"
+            class="border text-xs border-gray-100 rounded-md shadow w-10 justify-center
+          h-8 flex font-medium items-center py-1 px-2 bg-white cursor-pointer hover:bg-gray-100 transition mr-3 mb-4"
+            @click="switchOperator"
```

Comment:
If this is clicked and some filters are not applied, the not applied filters disappear

Reply:
fixed